clambcc/clambc-compiler.py

```diff
@@ -44,15 +42,28 @@
   -Wno-constant-conversion \
 "
 
+COMMON_WARNING_OPTIONS = [
+        "-Wno-backslash-newline-escape"
+      , "-Wno-pointer-sign"
+      , "-Wno-return-type"
+      , "-Wno-incompatible-pointer-types"
+      , "-Wno-unused-value"
+      , "-Wno-shift-negative-value"
+      , "-Wno-implicit-function-declaration"
+      , "-Wno-incompatible-library-redeclaration"
+      , "-Wno-implicit-int"
+      , "-Wno-constant-conversion"
```

Comment:
I would actually prefer to remove the warnings from the signature code, but since we don't maintain that, I think this is a good compermise.  I would like to see no warnings in any new signature code that is written.